src/main/java/org/openrewrite/java/micronaut/ChangeAnnotationProcessorPath.java

```diff
@@ -90,8 +92,15 @@ public String getDescription() {
 
     @Override
     public TreeVisitor<?, ExecutionContext> getVisitor() {
+
         return new MavenVisitor<ExecutionContext>() {
 
+            final DependencyMatcher depMatcher;
+
+            {
+                this.depMatcher = Objects.requireNonNull(DependencyMatcher.build(ChangeAnnotationProcessorPath.this.oldGroupId + ":" + ChangeAnnotationProcessorPath.this.oldArtifactId).getValue());
```

Comment:
Why the initialization with a block here? Just curious, although a clarification comment could help.

Reply:
No real reason other the example I was looking at (ChangeDependency maybe? Though now I can't seem to find it.) for use of DependencyMatcher had done it that way. :) I'll get rid of it.